share/commands/agent-history

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# @description read log of old agent runs
+# @man This command will output historic logs of agent runs.
+# @man +
+# @man *Options*:
+# @man +
+# @man *-c*: show history without color output
+# @man +
+# @man *-n*: show maximum n lines of history
+# @man +
+
+. "${BASEDIR}/../lib/common.sh"
+. "${BASEDIR}/../lib/cfengine_parsing.sh"
+
+BASE_PATH="/var/rudder/cfengine-community/outputs"
+# maximum 10000 lines
+max=25
```

Comment:
the link between the comment and the value is not obvious

Reply:
indeed